torch/_tensor_docs.py

```diff
@@ -2236,7 +2236,8 @@ def add_docstr_all(method, docstr):
     >>> x = torch.randn(3, 4, 5, device='cuda:0')
     >>> x.get_device()
     0
-    >>> x.cpu().get_device()  # RuntimeError: get_device is not implemented for type torch.FloatTensor
+    >>> x.cpu().get_device()
+    -1
```

Comment:
can you please also change the doc string at line 2232?

Reply:
sure. Done.